frontend/common/utils/errorHelpers.tsx

```diff
@@ -0,0 +1,17 @@
+import { t } from 'i18next';
+import { RequestError } from '../crud/RequestError';
+
+export function errorToMessage(error: unknown, messagePrefix?: string): string {
```

Comment:
hey @mabashian @lgalis there is an error adapter already parsing errors from EDA.

https://github.com/ansible/ansible-ui/blob/3e71e5a031b0293945405b71ae87c73bec3eb29a/frontend/eda/common/edaErrorAdapter.tsx#L8

Also https://github.com/ansible/ansible-ui/blob/3e71e5a031b0293945405b71ae87c73bec3eb29a/frontend/eda/common/edaErrorAdapter.tsx#L45

We might able to reuse those to parse errors here.